api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -990,9 +925,103 @@ public void test_getRegisteredTierExpirations_one_year() {
     assertThat(expirations.get(0).getExpirationDate()).isEqualTo(aYearFromNow);
   }
 
+  // regression test: confirm that expirations are returned when the compliance module is disabled
+  // and the user has not completed or bypassed compliance
+
+  @Test
+  public void test_getRegisteredTierExpirations_one_year_compliance_disabled() {
+    providedWorkbenchConfig.access.enableComplianceTraining = false;
+
+    // register user by setting the DUCC bypass to now
+    // and the 2 unbypassable modules' completions to now
+
+    final Timestamp now = Timestamp.from(PROVIDED_CLOCK.instant());
+    dbUser =
+        updateUserWithRetries(
+            user -> {
+
+              // this is sufficient to fully register the user when the compliance module is
+              // disabled
+
+              user.setDisabled(false);
+              user.setEraCommonsBypassTime(now);
+              user.setTwoFactorAuthBypassTime(now);
+              user.setDataUseAgreementBypassTime(now);
+              user.setPublicationsLastConfirmedTime(now);
+              user.setProfileLastConfirmedTime(now);
+
+              // ensure there is nothing set for compliance
+
+              user.setComplianceTrainingCompletionTime(null);
+              user.setComplianceTrainingBypassTime(null);
+
+              return user;
+            });
+    assertRegisteredTierEnabled(dbUser);
+
+    // the 2 unbypassable modules will expire in a year
+    final String aYearFromNow =
+        PROVIDED_CLOCK.instant().plus(EXPIRATION_DAYS, ChronoUnit.DAYS).toString();
+
+    final List<UserAccessExpiration> expirations = userService.getRegisteredTierExpirations();
+    assertThat(expirations.size()).isEqualTo(1);
```

Comment:
This line is the Preprod blocker: before the fix, the report would show no expirations.